apps/ledger-live-mobile/src/components/ServicesWidget/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { Flex, Text } from "@ledgerhq/native-ui";
+import { protectContext } from "@ledgerhq/live-common/platform/providers/ProtectProvider/index";
+import { ProtectStateNumberEnum } from "@ledgerhq/live-common/platform/providers/ProtectProvider/types";
+import React, { memo, useContext } from "react";
+import { useTranslation } from "react-i18next";
+import Svg, { LinearGradient, Defs, Rect, Stop } from "react-native-svg";
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import NewProtectState from "./Protect/NewProtectState";
+import ConfirmIdentityProtectState from "./Protect/ConfirmIdentityProtectState";
+import AddPaymentProtectState from "./Protect/AddPaymentProtectState";
+import SubscriptionCanceledProtectState from "./Protect/SubscriptionCanceledProtectState";
+import PaymentRejectedProtectState from "./Protect/PaymentRejectedProtectState";
+import ActiveProtectState from "./Protect/ActiveProtectState";
+import { ServicesConfig } from "./types";
+
+const SvgGradient = () => (
+  <Svg width="100%" height="8px">
+    <Defs>
+      <LinearGradient
+        id="protectGradient"
+        x1="0%"
+        y1="0%"
+        x2="100%"
+        y2="0%"
+        gradientUnits="userSpaceOnUse"
+      >
+        <Stop offset="0%" stopOpacity={1} stopColor="hsla(40, 90%, 69%, 1)" />
```

Comment:
Are you sure that it works on light/dark mode ?

Reply:
same here